src/sql/src/plan/statement/ddl.rs

```diff
@@ -272,6 +272,13 @@ pub fn plan_create_table(
 
     let names: Vec<_> = columns
         .iter()
+        .filter(|c| {
+            let is_versioned = c
+                .options
+                .iter()
+                .any(|o| matches!(o.option, ColumnOption::Versioned { .. }));
+            !is_versioned
```

Comment:
Why are we filtering here?

Reply:
Added a comment, but it's because of how the `names` collection is used, I took a note for myself to refactor this entire block